pkg/cmd/kafka/acl/list/list.go

```diff
@@ -116,7 +116,7 @@ func NewListACLCommand(f *factory.Factory) *cobra.Command {
 	flags := flagutil.NewFlagSet(cmd, f)
 
 	flags.AddInstanceID(&opts.kafkaID)
-	flags.AddOutput(&opts.output)
+	flags.AddOutput(&opts.output, cobra.ShellCompDirectiveNoSpace)
```

Comment:
Please ignore this change, this was a typo/error which i was to delete before upload